.env.tpl

```diff
@@ -19,6 +19,7 @@ AUTH_SECRET=keycloakZaakafhandelcomponentClientSecret
 AUTH_SERVER=http://localhost:8081
 BAG_API_CLIENT_MP_REST_URL=op://Dimpact/ZAC-.env-$APP_ENV/BAG/API_CLIENT_MP_REST_URL
 BAG_API_KEY=op://Dimpact/ZAC-.env-$APP_ENV/BAG/API_KEY
+BRON_ORGANISATIE=123443210
```

Comment:
We should also add these new env vars to our `.env.example` with a description of what these vars are. This because the env.example file is our documentation of the ZAC env vars.

Reply:
Added